### PR TITLE
Write unit test

### PR DESCRIPTION
**Pull request by hungnd-3027**
## WHAT
- Write unit test
## Review Checklist
image
image
image
image
image

Category | View Point | Description | Self review | Reviewer2 (name)
--- | --- | --- | --- | ---
Conventions | Sun* coding conventions | https://sal.vn/WtYPAy |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Conventions | Kotlin coding conventions | https://kotlinlang.org/docs/coding-conventions.html |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Sun* Redmine working process  | https://sal.vn/piPoQn |<li>- [ ] yes</li>|<li>- [ ] yes</li>  

## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

**Comment by ghost** [comment]
chụp ảnh kết quả coverage sau khi test lên đi ông

**Comment by hungnd-3027** [comment]
> chụp ảnh kết quả coverage sau khi test lên đi ông

oke ông

**Review by ghost — APPROVED** [review]

File: app/src/test/java/com/example/cooktaildb/ui/detail/DetailDrinkActivityPresenterTest.kt [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+            repository.isFavorite(idDrink, capture(localCallbackBoolean))
+        } answers {
+            localCallbackBoolean.captured.onFailed()
+        }
+        detailDrinkPresenter.isFavorite(idDrink)
+        verify {
+            view.showError()
+        }
+    }
+}
```
Comment (ghost) [inline app/src/test/java/com/example/cooktaildb/ui/detail/DetailDrinkActivityPresenterTest.kt]: endline này

Comment (hungnd-3027) [reply app/src/test/java/com/example/cooktaildb/ui/detail/DetailDrinkActivityPresenterTest.kt]: oke ông, để t đẩy lại

**Review by vietnh-1109 — APPROVED** [review]

**Pull request merged**